app/views/content/explore-teaching-advisers/_article.html.erb

```diff
@@ -12,7 +12,52 @@
       large: true
     ) %>
   </section>
+</div>
```

Comment:
Just a reminder that this change will need removing from this PR before it is merged @spencerldixon @jenhadfield-dfe !